test/Cpp/src/multiport/IndexIntoMultiportInput.lf

```diff
@@ -0,0 +1,46 @@
+target Cpp
+
```

Comment:
I think it's a good practice to add a description to the test so that it's clear to others what the point of the test is.